src/mechanics/tools/TimeDependentProblem.cpp

```diff
@@ -0,0 +1,86 @@
+#include "mechanics/tools/TimeDependentProblem.h"
+#include "mechanics/dofs/DofNumbering.h"
+
+using namespace NuTo;
+
+TimeDependentProblem::TimeDependentProblem(MeshFem* rMesh)
```

Comment:
Why does Time dependent problem does only work for FEM? IMO, there should not be a difference between IGA and FEM when it comes to the solution procedure.